tests/release.yaml

```diff
@@ -0,0 +1,12 @@
+# Contains all operators required to run the test suite.
+---
+releases:
+  # Do not change the name of the release as it's referenced from run_tests.sh
+  tests:
+    releaseDate: 1970-01-01
+    description: Integration test
+    products:
+      commons:
+        operatorVersion: 0.0.0-dev
+      listener:
+        operatorVersion: 0.0.0-dev
```

Comment:
do we need the secret operator here as well?

Reply:
> do we need the secret operator here as well?

The test doesn't  need it